www/form/inner.js

```diff
@@ -5099,9 +5136,30 @@ define([
                 editableStr
             ]);
 
+            var toggleOffclass = 'ontouchstart' in window ? 'toggle-active' : 'toggle-inactive'; 
+            var toggleOnclass = 'ontouchstart' in window ? 'toggle-inactive' : 'toggle-active'; 
+            var toggleDragOff = h(`button#toggle-drag-off.cp-form-view-drag.${toggleOffclass}.fa.fa-arrows`, {'aria-hidden': true, 'title': Messages.toggleArrows});
+            var toggleDragOn = h(`button#toggle-drag-on.cp-form-view-drag.${toggleOnclass}.fa.fa-hand-o-up`, {'aria-hidden': true, 'title': Messages.toggleDrag});
+            $(toggleDragOff).click(function() {
+                $(toggleDragOff).attr('class').indexOf('toggle-inactive') !== -1 ? $(toggleDragOff).toggleClass('toggle-inactive').toggleClass('toggle-active') && $(toggleDragOn).toggleClass('toggle-active').toggleClass('toggle-inactive') : undefined;
```

Comment:
You should try as much as possible to avoid long lines. Ideally, they should be shorter than 80 characters. In this case, using a ternary with an "undefined" at the end makes it all very difficult to read instead of a simple `if` statement.
I also don't think you need two classes here. If a button doesn't have the "active" class, it means it's "inactive"!